src/components/settings/Config.tsx

```diff
@@ -123,20 +129,24 @@ const Config = () => {
           sidetitle={<DisconnectButton />}
           subsidetitle={
             <ButtonToolbar>
-              <StyledButton
-                size="sm"
-                onClick={async () => {
-                  apiController({
-                    serverType: config.serverType,
-                    endpoint: 'startScan',
-                    args: { musicFolderId: folder.musicFolder },
-                  });
-                  setIsScanning(true);
-                }}
-                disabled={isScanning}
-              >
-                {isScanning ? `${scanProgress}` : 'Scan'}
-              </StyledButton>
+              {useAppSelector((state) => state.config).serverType !== 'funkwhale' && (
```

Comment:
`config` is already defined in this file, so you don't need to inline the `useAppSelector` call again. Also use the `Server` enum.